### PR TITLE
Data imputation

### PR DESCRIPTION
**Pull request by YeWenting**
- process the missing value
- speed up the develop employment

### CODE REVIEW

File: src/frontend/src/components/GMMatrixVisualization2/GMMatrixVisualization2.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ export default class GMMatrixVisualization2 extends PureComponent {
         .then(res => {
             res.json()
             .then (json => {
-                //console.log(json)
+                // console.log(json)
```
Comment (blengerich) [inline src/frontend/src/components/GMMatrixVisualization2/GMMatrixVisualization2.js]: please remove commented-out code in this file

File: src/IO/PlinkReader.cpp [hunk 2]
```diff
@@ -75,17 +74,17 @@ void PlinkReader::getX(string filename, MatrixXf &X)
     long m = pio_num_samples( &plink_file );
     long n = pio_num_loci(&plink_file );
```
Comment (blengerich) [inline src/IO/PlinkReader.cpp]: should this always print?

Comment (lebronlambert) [reply src/IO/PlinkReader.cpp]: We should ask wenting about that one~ I will tell him

File: src/frontend/src/components/GMMatrixVisualization2/GMMatrixVisualization2.js (hunk 1 above)
Comment (YeWenting) [reply src/frontend/src/components/GMMatrixVisualization2/GMMatrixVisualization2.js]: I have deleted this code.

File: src/IO/PlinkReader.cpp (hunk 2 above)
Comment (YeWenting) [reply src/IO/PlinkReader.cpp]: The output only shows on the server, but you're right, it can be deleted.

**Pull request merged**